### PR TITLE
random code

### PR DESCRIPTION
**Pull request by bakito**

### CODE REVIEW

File: eastercamp2021/assets/StartReaktor [hunk 1]
```diff
@@ -1,12 +1,10 @@
 #!/bin/bash
 
-code=331214
-
 echo 0 > /tmp/reaktor
 
 read -p "Code: " code
 
-if [ $((code % 2)) -eq 0 ]; then
+if [[ "$code" == "RANDOM_CODE" ]]; then
```
Comment (snorwin) [inline eastercamp2021/assets/StartReaktor]: `$` is missing

Comment (bakito) [reply eastercamp2021/assets/StartReaktor]: No, this is sed replaced configure.sh

Comment (snorwin) [reply eastercamp2021/assets/StartReaktor]: ah true, sorry I missed that

File: eastercamp2021/configure.sh [hunk 2]
```diff
@@ -37,4 +37,9 @@ echo "function kill() { /bin/ps -p \$1 -o command > /tmp/kill && /bin/kill "\$@"
...
 echo "cd /AmongUs/Cafeteria" >> /root/.bashrc
 
-echo 'done' > /opt/katacoda-background-finished
\ No newline at end of file
+CODE=$( ((RND=RANDOM<<15|RANDOM)) ; echo ${RND: -6})
+sed -i "s/@RANDOM_CODE@/$CODE/g" /AmongUs/Cafeteria/Admin/Antrieb/Code
+
+echo 'done' > /opt/katacoda-background-finished
+
+U+1F5E1
```
Comment (snorwin) [inline eastercamp2021/configure.sh]: not sure if this unicode char is needed here

**Pull request merged**